src/cart/utils/crt_utils.h

```diff
@@ -13,42 +13,39 @@
 #include "../crt_internal.h"
 #include "svc.pb-c.h"
 
-#define DBG_PRINT(x...)							\
-	do {								\
-		struct test_options *__opts;				\
-		__opts = crtu_get_opts();					\
-		D_INFO(x);						\
-		if (__opts->is_server)					\
-			fprintf(stderr, "SRV [rank=%d pid=%d]\t",       \
-			__opts->self_rank,				\
-			__opts->mypid);					\
-		else							\
-			fprintf(stderr, "CLI [rank=%d pid=%d]\t",       \
-			__opts->self_rank,				\
-			__opts->mypid);					\
-		fprintf(stderr, x);					\
+#define DBG_PRINT(x...)                                                                            \
+	do {                                                                                       \
+		struct test_options *__opts;                                                       \
+		__opts = crtu_get_opts();                                                          \
+		D_INFO(x);                                                                         \
+		if (__opts->is_server)                                                             \
+			fprintf(stderr, "SRV [rank=%d pid=%d]\t", __opts->self_rank,               \
+				__opts->mypid);                                                    \
+		else                                                                               \
+			fprintf(stderr, "CLI [rank=%d pid=%d]\t", __opts->self_rank,               \
+				__opts->mypid);                                                    \
+		fprintf(stderr, x);                                                                \
 	} while (0)
 
 struct test_options {
-	bool		is_initialized;
-	d_rank_t	self_rank;
-	int		mypid;
-	int		num_attach_retries;
-	bool		is_server;
-	bool		assert_on_error;
-
-	volatile int	shutdown;
-	int		delay_shutdown_sec;
-	bool		is_swim_enabled;
-	bool		use_daos_agent_env;
-
+	bool         is_initialized;
+	d_rank_t     self_rank;
+	int          mypid;
+	int          num_attach_retries;
+	bool         is_server;
+	bool         assert_on_error;
+
+	volatile int shutdown;
```

Comment:
(style)  Use of volatile is usually wrong: see Documentation/volatile-considered-harmful.txt